packages/headless/src/controllers/smart-snippet/headless-smart-snippet.ts

```diff
@@ -99,13 +128,14 @@ export function buildSmartSnippet(engine: SearchEngine): SmartSnippet {
     get state() {
       const state = getState();
       return {
-        disliked: state.questionAnswering.disliked,
-        liked: state.questionAnswering.liked,
-        expanded: state.questionAnswering.expanded,
-        answerFound: state.search.response.questionAnswer.answerSnippet !== '',
```

Comment:
These parts of the state weren't removed, I reordered them to match the order of their definition in the SmartSnippetState interface.